bench/bench.ml

```diff
@@ -491,6 +528,11 @@ let json_flag =
   let env = env_var "JSON" in
   Arg.(value & flag & info [ "j"; "json" ] ~env ~doc)
 
+let frequency =
+  let doc = "The frequency of timing the replace operations." in
+  let env = env_var "FREQ" in
+  Arg.(value & opt int 10 & info [ "freq" ] ~env ~doc)
```

Comment:
This flag (and its corresponding `env_var` seem a bit too generically named). Although `REPLACE_DURATION_SAMPLING_INTERVAL` might be too verbose :wink: